securedrop/tests/test_crypto_util.py

```diff
@@ -19,6 +19,20 @@ def setUp(self):
     def tearDown(self):
         utils.env.teardown()
 
+    def test_word_lists_imports(self):
+        """Make sure word count for each list is correct and Guard
+        against regressions regarding not stripping file contents
+        correctly.
+        """
+        words = crypto_util.words
+        nouns = crypto_util.nouns
+        adjectives = crypto_util.adjectives
+
+        self.assertEqual(len(words), 7603)
+        self.assertEqual(len(adjectives), 8222)
+        self.assertEqual(len(nouns), 17949)
```

Comment:
Let's remove the asserts in lines 31-33 and keep just the assert below on line 34 - your assert in line 34 is a nice clean way to catch any null strings making it into these word lists in the future. You might also rename the test `test_word_lists_contain_no_null_strings`. (I say this because we've changed our word lists several times and for maintainability's sake, we don't want to edit this unit test every time we do that especially when these asserts are unnecessary to catch the bug in question).